web/src/settings_config.ts

```diff
@@ -129,6 +129,14 @@ export const web_home_view_values = {
     },
 };
 
+type ColorScheme = "automatic" | "dark" | "light";
+export type ColorSchemeDetails = {
+    [key in ColorScheme]: {
+        code: number;
+        description: string;
+    };
+};
+
 export const color_scheme_values = {
```

Comment:
Is `color_scheme_values` supposed to be marked as of the type `ColorSchemeDetails`?